extensions/Bilibili/src/utils/api.ts

```diff
@@ -1,13 +1,18 @@
+const BASE_PASSPORT_URL = "http://passport.bilibili.com";
```

Comment:
https?

Reply:
Oh, this API supports both HTTP and HTTPS. I’ll change it later.